ddtrace/tracer.py

```diff
@@ -28,6 +28,18 @@
 
 log = get_logger(__name__)
 
+debug_mode = asbool(get_env("trace", "debug", default=False))
+
+DD_LOG_FORMAT = "%(asctime)s %(levelname)s [%(name)s] [%(filename)s:%(lineno)d] {}- %(message)s".format(
+    "[dd.service=%(dd.service)s dd.env=%(dd.env)s dd.version=%(dd.version)s"
+    " dd.trace_id=%(dd.trace_id)s dd.span_id=%(dd.span_id)s] "
+)
+if debug_mode and not hasHandlers(log):
```

Comment:
we could also explicitly set the root `logging.getLogger("ddtrace")` loggers level to debug as well... but we'd have to add a handler too if one wasn't defined as well.

Reply:
Hmm outside the scope of this PR?

Reply:
yes, outside the scope, but wasn't sure which change was easier/safer to introduce, your call here